docs/modules/ROOT/pages/servlet/test/method.adoc

```diff
@@ -61,13 +61,13 @@ public class WithMockUserTests {
 .Kotlin
 [source,kotlin,role="secondary"]
 ----
-@RunWith(SpringJUnit4ClassRunner::class)
+@ExtendWith(SpringExtension.class)
 @ContextConfiguration
 class WithMockUserTests {
     // ...
 }
 ----
-<1> `@RunWith` instructs the spring-test module that it should create an `ApplicationContext`. This is no different than using the existing Spring Test support. For additional information, refer to the https://docs.spring.io/spring-framework/docs/4.0.x/spring-framework-reference/htmlsingle/#integration-testing-annotations-standard[Spring Reference]
+<1> `@ExtendWith` instructs the spring-test module that it should create an `ApplicationContext`.
```

Comment:
I think about linking [Spring docs](https://docs.spring.io/spring-boot/docs/current/reference/html/features.html#features.testing.spring-boot-applications) or [Junit 5 extensions](https://junit.org/junit5/docs/current/user-guide/#extensions)

Reply:
Yes, I think that you might consider linking `@ExtendWith` to where it's discussed in the Spring Test documentation: https://docs.spring.io/spring-framework/docs/current/reference/html/testing.html#testcontext-junit-jupiter-extension. I'd also recommend that you use the `springFrameworkReferenceUrl` variable instead of hardcoding the version.